spec/features/openid_connect/openid_connect_spec.rb

```diff
@@ -3,6 +3,7 @@
 describe 'OpenID Connect' do
   include IdvHelper
   include CloudhsmMocks
+  include RememberDeviceConcern
```

Comment:
It looks like we only added `uncheck` here. Was this left behind from trying a different approach or am I missing something?

Reply:
Woops yes it was left behind, thanks for the catch!